x/deployment/types/types.go

```diff
@@ -65,6 +65,15 @@ var GroupStateMap = map[string]GroupState{
 	"closed":       GroupClosed,
 }
 
+// GroupStateToReadable returns the adverb of the state
+func GroupStateToReadable(g GroupState) string {
+	m := make(map[GroupState]string, len(GroupStateMap))
+	for k, v := range GroupStateMap {
+		m[v] = k
+	}
+	return m[g]
```

Comment:
Let's just use [stringer](https://godoc.org/golang.org/x/tools/cmd/stringer).

with stringer this is moot, but you don't need to make complete copy.

```go
for k, v := range GroupStateMap {
  if v == g {
    return k
  }
}
return "UNKNOWN"
```

Reply:
Updated in code, an issue created: https://github.com/ovrclk/akash/issues/665